firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -30,7 +35,7 @@
    * Checks if the FIS Auth token is expired or going to expire in next 1 hour {@link
    * #AUTH_TOKEN_EXPIRATION_BUFFER_IN_SECS}.
    */
-  public boolean isAuthTokenExpired(PersistedInstallationEntry entry) {
+  public boolean isAuthTokenExpired(@NonNull PersistedInstallationEntry entry) {
```

Comment:
static?

Reply:
https://github.com/firebase/firebase-android-sdk/pull/1954/files#r487182669

Reply:
a) Please add a respective comment to the code.
b) Just to be sure we are on the same page:
If this class were to use a proper `TimeSource`, this issue would not exist and tests could mock the TimeSource instance instead.

Reply:
a) Done. 
b) Correct.